src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -0,0 +1,100 @@
+import path from 'path';
+
+Cypress.Commands.add('checks', () => {
```

Comment:
How about making this command more descriptive, e.g. `checkSearch`?

Reply:
Also, does `Cypress.Commands.add` affect the global namespace for all Cypress tests, or just the file in which the command appears?

If it's the former, we should probably add a namespace prefix, e.g. `facilities.checkSearch`.

Reply:
https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax

Reply:
OK I was able to answer my own question. The answer appears to be that `Cypress.Commands.add()` only affects the current file, so there is no need to worry about namespacing.

To confirm this, I added a call to `cy.checks()` in the search test, then ran the mobile and search tests sequentially using the following command:

`❯ npx cypress run --config-file  config/cypress.json --spec src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js,src/applications/facility-locator/tests/e2e/search.cypress.spec.js`

I got this error:

`TypeError: cy.checks is not a function`